scripts/get_latest_blockhashes.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import logging
+import json
+from pathlib import Path
+from argparse import ArgumentParser
+
+from starkware.starknet.services.api.feeder_gateway.feeder_gateway_client import FeederGatewayClient
+from services.external_api.client import RetryConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+NETWORKS = {
+    "goerli": "https://alpha4.starknet.io/feeder_gateway", # alpha-goerli
+    "mainnet": "https://alpha-mainnet.starknet.io/feeder_gateway/" # mainnet
+}
+
+
+parser = ArgumentParser(description="Get block information from sequencer")
+parser.add_argument("--network", "-n", default="goerli", type=str, help=f"Select network, one of {list(NETWORKS.keys())}")
+args = parser.parse_args()
+
+async def main():
+    # Instantiate a FeederGatewayClient object
+    # -1 means unlimited retries
+    retry_config = RetryConfig(n_retries=-1)
+    feeder_gateway_client = FeederGatewayClient(url=NETWORKS[args.network], retry_config=retry_config)
+
+    # Get the latest block
+    # Sometimes get_block returns "null" as value
+    # Walrus operator for assignment expression
+    while (latest_block := await feeder_gateway_client.get_block()) == "null":
+        continue
+    logger.info(f"Latest block number: {latest_block.block_number}")
+
+    # Get the last 256 blocks excluding the current block
+    # Convert the list of blockhashes in to dictionary { block_number: block_hash }
+    last_256_blocks = [
+        await feeder_gateway_client.get_block(block_number=latest_block.block_number - i)
+        for i in range(1, 257)
+    ]
+    last_256_blockhashes = {
+        block.block_number: block.block_hash
+        for block in last_256_blocks
+    }
+    
+    # Dump JSON to file
+    with open(Path("sequencer") / "mock_blockhashes.json", "w") as file:
```

Comment:
Why are we still dumping in JSON?

Reply:
Thanks for the review Elias! Great question, we still need to create this mock file mostly for testing. This will be passed in for when the fixtures that sets the `Starknet` block info and initializes the block hash registry. I think fetching the latest block hash every time we test would add to the time so using this mock file can be handy.